pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
+
+- Download file using remote name:
+
+`curl -O {{http://example.com/filename}}`
+
+- Automatically continue/resume previous file transfer:
+
+`curl -O -C - {{http://example.com/filename}}`
 
 - Send form-encoded data:
 
-`curl --data {{name=bob}} {{http://localhost/form}}`
+`curl --data {{name=bob}} {{http://example.com/form}}`
 
 - Send JSON data:
 
-`curl -X POST -H "Content-Type: application/json" -d {{'{"name":"bob"}'}} {{http://localhost/login}}`
+`curl -X POST -H "Content-Type: application/json" -d {{'{"name":"bob"}'}} {{http://example.com/login}}`
 
 - Specify an HTTP method:
 
-`curl -X {{DELETE}} {{http://localhost/item/123}}`
+`curl -X {{DELETE}} {{http://example.com/item/123}}`
 
 - Head request:
```

Comment:
@waldyrious - They are indeed different. `--head` will print only the response headers. This is usually achieved with the HEAD http method. But you can also use the `--head` option with GET method, in which case it ignores the response body and only prints the headers. This is kind of unecessary IMO.

I would either do `curl -X HEAD http://example.com` or `curl http://example.com -v -o /dev/null`.